swagger.yaml

```diff
@@ -923,6 +923,47 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  "/severa/users/{severaUserId}/workWeek":
+    get:
+      operationId: getUserContractedWorkWeek
```

Comment:
This could be just:
getUserWorkWeekLength? Maybe clearer and maybe the contracted is unnecessary- this could be true in other places if you agree?